app/Http/Requests/CreateEventRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Http\Requests\Request;
+
+class CreateEventRequest extends Request
+{
```

Comment:
it will **EventRequest** because it will handle the validation for creation and update